### PR TITLE
Fixed #1956 by implementing shield textures, animations and properties

### LINKED ISSUE #1956: [1.18.2] Shield Textures Messed Up
Some shield textures are screwed up and none of the shields are held correctly in the hand.

### PR DESCRIPTION
**Pull request by neumann-lukas**
Changes over 1.12:
- Copied shield model jsons from 1.12
- added `"gui_light": "front",` to all model jsons
- Implemented ` ItemProperties.register` in the Item init. #1956 

Therefor fixed #1956 .

### CODE REVIEW

File: src/main/java/net/dries007/tfc/common/items/TFCShieldItem.java [hunk 1]
```diff
@@ -18,6 +21,9 @@ public TFCShieldItem(Tier tier, Properties builder)
     {
         super(builder.defaultDurability(tier.getUses()));
         this.tier = tier;
+        if (FMLEnvironment.dist.isClient()) {
+            ItemProperties.register(this, new ResourceLocation("blocking"), (stack, world, living, i) -> living != null && living.isUsingItem() && living.getUseItem() == stack ? 1.0f : 0.0f);
+        }
```
Comment (alcatrazEscapee) [inline src/main/java/net/dries007/tfc/common/items/TFCShieldItem.java]: Prefer grouping this with our other registrations of this type in [ClientEventHandler](https://github.com/TerraFirmaCraft/TerraFirmaCraft/blob/5750590698c490b91d4a1526cf9e866b59836264/src/main/java/net/dries007/tfc/client/ClientEventHandler.java#L145-L176)

File: src/main/resources/assets/tfc/models/item/metal/shield/bismuth_bronze.json [hunk 2]
```diff
@@ -1,7 +1,409 @@
...
+    "2": "tfc:item/metal/shield/bismuth_bronze_back",
+    "particle": "tfc:item/metal/shield/bismuth_bronze_front"
+  },
+  "elements": [
+    {
+      "name": "h",
+      "from": [7, 10, 8],
+      "to": [8, 11, 13],
+      "faces": {
+        "north": {"uv": [0, 0, 0, 0], "texture": "#texture"},
```
Comment (alcatrazEscapee) [inline src/main/resources/assets/tfc/models/item/metal/shield/bismuth_bronze.json]: The `#missing` textures could probably be removed from the models entirely here.

**Review by alcatrazEscapee — CHANGES_REQUESTED** [review]
Thank you very much for the contribution! I just have two suggestions, the latter seems like it applies to most/all the models included.

**Comment by neumann-lukas** [comment]
Should be finished now :-)

**Pull request merged**